exasol_script_languages_developer_sandbox/lib/aws_access.py

```diff
@@ -39,7 +44,7 @@ def delete_ec2_key_pair(self, key_name: str) -> None:
         cloud_client = self._get_aws_client("ec2")
         cloud_client.delete_key_pair(KeyName=key_name)
 
-    def upload_cloudformation_stack(self, yml: str, stack_name: str):
+    def upload_cloudformation_stack(self, yml: str, stack_name: str, tags=tuple()):
```

Comment:
would have guessed tags is a dict

Reply:
The typical parameter in this project is:
```
({"Key": "exa_slc_id", "Value": "5.0.0"})
```

So, it's a tuple of dicionaries.